CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 
 ## New Features
 
-- PR #455 CSV Reader: Add support for user-specified decimal point and thousands separator
+- PR #455 #501 CSV Reader: add support for user-specified decimal point and thousands separator
```

Comment:
Unsure if you can easily, but would be better to split this into two separate lines explaining what each PR did separately.